oauth2_http/javatests/com/google/auth/oauth2/ExternalAccountAuthorizedUserCredentialsTest.java

```diff
@@ -301,13 +333,35 @@ public void toBuilder() {
             .setRevokeUrl(REVOKE_URL)
             .setAccessToken(new AccessToken(ACCESS_TOKEN, new Date()))
             .setQuotaProjectId(QUOTA_PROJECT)
+            .setUniverseDomain(UNIVERSE_DOMAIN)
             .build();
 
     ExternalAccountAuthorizedUserCredentials secondCredentials = credentials.toBuilder().build();
 
     assertEquals(credentials, secondCredentials);
```

Comment:
can you also add a test for universeDomain equals? like for other fields? equals_differentUniverseDomain()

This check base implementation is called